import_tracker/import_tracker.py

```diff
@@ -419,15 +455,23 @@ def _get_imports(mod: ModuleType) -> Tuple[Set[ModuleType], Set[ModuleType]]:
     open_import = False
     open_tries = set()
     log.debug4("Byte Code:")
-    for line in bcode.dis().split("\n"):
+    dis_lines = bcode.dis().split("\n")
```

Comment:
I don't love the need to iterate the lines twice, but this should be a pretty small performance penalty